cmd/mtaBuild.go

```diff
@@ -188,8 +188,23 @@ func runMtaBuild(config mtaBuildOptions,
 
 	commonPipelineEnvironment.mtarFilePath = mtarName
 
+	// install maven artifacts in local maven repo because `mbt build` executes `mvn package -B`
 	err = installMavenArtifacts(e, config)
+	// mta-builder executes 'npm install --production', therefore we need 'npm ci/install' to install the dev-dependencies
+	err = installNpmDependencies()
+
+	return err
+}
 
+func installNpmDependencies() error {
+	npmExecutor, err := npm.NewExecutor(npm.ExecutorOptions{Install: true})
+	if err != nil {
+		return err
+	}
+	if npmExecutor != nil {
+		x := npmExecutor.FindPackageJSONFiles()
+		err = npmExecutor.InstallAllDependencies(x)
+	}
 	return err
```

Comment:
After changing the signature of `npm.NewExecutor()`, this should then be:
```suggestion
	npmExecutor := npm.NewExecutor(npm.ExecutorOptions{Install: true})
	packageJSONFiles := npmExecutor.FindPackageJSONFiles()
	return npmExecutor.InstallAllDependencies(packageJSONFiles)
```